docs/course/index.md

```diff
@@ -130,11 +130,7 @@ This file specifies basic information about the course:
     }
     ```
 
-??? note "Format specification for `infoCourse.json`"
-
-    ```json
-    --8<-- "apps/prairielearn/src/schemas/schemas/infoCourse.json"
-    ```
+!!! info "[`infoCourse.json` reference](../schemas/infoCourse.md) :robot:"
```

Comment:
For lack of a better place to put this comment, the schema is wrong:

<img width="831" alt="Screenshot 2025-03-27 at 14 11 42" src="https://github.com/user-attachments/assets/6f19ab85-e5fa-46a5-bf17-85819eab685f" />

Topics don't support a `shortName`, and "assessment sets" shouldn't be mentioned in this context. The latter is easy to fix. Let me see if I can validate that all existing courses don't have `shortName` in their topics; if any do, removing that would be a breaking change.

@mwest1066 _are_ topics visible to students? I can't think of any place we show these to them.

Reply:
I checked; `shortName` doesn't appear in any `infoCourse.json` files, so we're clear to remove that from the topics schema.

Reply:
It also doesn't look like it's part of `TopicSchema` in `db-types`.

Reply:
It also doesn't look like it's part of `TopicSchema` in `db-types`.

Reply:
Thanks for changing that. Looks like you skipped my comment about the phrase "assessment sets" incorrectly showing up under the topics schema?

Reply:
Whoops, I missed that. Changed in a38f4701622378a716082248864d2cefc599afa9 to `"A single topic, can represent a unit of learning (e.g. 'vectors').`